### PR TITLE
APIT section & Product menu

### PR DESCRIPTION
**Pull request by yashwin**

### CODE REVIEW

File: _assets/js/main.js [hunk 1]
```diff
@@ -500,6 +500,90 @@ ListAnimator.prototype.binder = function(Method){
...
+  }catch(e){}
+}
+
+////////////////////////////////////////////////////////
+//Method to prevent navigation of product-link in mobile menu
+function productLinkClickAction(){
+  $("#product-link").click(function(ev){
+    var doc = $(document);
+    if(doc.width()<760){
+      ev.preventDefault;
```
Comment (subho007) [inline _assets/js/main.js]: __[JSHint]__ Expected an assignment or function call and instead saw an expression.

<sub>posted by Sider</sub>

File: _assets/js/main.js [hunk 2]
```diff
@@ -500,6 +500,90 @@ ListAnimator.prototype.binder = function(Method){
...
+    }else{
+      document.location = ev.currentTarget.href;
+    }
+  })
```
Comment (subho007) [inline _assets/js/main.js]: __[JSHint]__ Missing semicolon.

<sub>posted by Sider</sub>

File: _assets/css/main.css [hunk 3]
```diff
@@ -2125,12 +2125,17 @@ ul.pricing-plan-list {
   padding-top: 130px
 }
 
+.section-leveler {
+  margin-top: -2em!important;
```
Comment (subho007) [inline _assets/css/main.css]: I hate `!important` @cosmosgenius @Praseetha-KR thoughts?

**Review by subho007 — CHANGES_REQUESTED** [review]

File: _assets/css/main.css (hunk 3 above)
Comment (cosmosgenius) [reply _assets/css/main.css]: if possible remove!

Comment (yashwin) [reply _assets/css/main.css]: oh shit, removed it 👍

**Review by subho007 — APPROVED** [review]

**Pull request merged**